src/main/java/Duke.java

```diff
@@ -1,10 +1,221 @@
+//Matric Number: A0200273X
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        showWelcomeMessage();
+        //String Array of size 100 created to store item list
+        String[] itemList = new String[100];
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        Duke numTasks =new Duke();
```

Comment:
I'm not sure of the purpose of this line. Perhaps clearer variable naming or a comment would be appropriate here?